docs/kyma-environment-broker/03-10-orchestration.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Orchestration
+type: Details
+---
+
+Orchestration is a mechanism that allows you to upgrade Kyma Runtimes. To create an orchestration, [send the upgrade request to the proper KEB handler](#tutorials-upgrade-kyma-runtime-using-keb). After sending the request, the orchestration is processed by `KymaUpgradeManager`. It lists Shoots (Kyma Runtimes) in the Gardener cluster and filters the Runtime IDs that you have specified for the upgrade in the request body. Then, the `KymaUpgradeManager` proceeds with the orchestration and performs the [upgrade steps](#details-runtime-operations) logic on the Runtimes that you have specified.
+
+>**NOTE:** You need a token with the `broker-upgrade:write` authorization scope to create an orchestration, and a token with the `broker-upgrade:read` scope to fetch the orchestrations.
+
+>**NOTE:** For now, you can upgrade only one Runtime in a single request in KEB.
+
+To check the status of the orchestration, use one of the `GET` endpoints.
+Use Swagger to [check how to consume the Kyma Orchestration API](https://app.swaggerhub.com/apis/kempski/kyma-orchestration_api/0.1).
```

Comment:
I'd consider describing the API in more detail in this doc.

Reply:
imo we can - list our API's endpoints and briefly describe them, and then we can link to the Swagger for more details.

Reply:
I like this idea 🚀